src/ExcelFiles.jl

```diff
@@ -22,11 +22,21 @@ function Base.show(io::IO, ::MIME"text/html", source::ExcelFile)
 end
 
 Base.Multimedia.showable(::MIME"text/html", source::ExcelFile) = true
-
 function fileio_load(f::FileIO.File{FileIO.format"Excel"}, range; keywords...)
     return ExcelFile(f.filename, range, keywords)
 end
 
+function fileio_save(f::FileIO.File{FileIO.format"Excel"}, data; keywords...)
```

Comment:
I think you can just make this:
````julia
function fileio_save(f::FileIO.File{FileIO.format"Excel"}, data; sheetname="Sheet1")
````
and it should all work? That is how I did keyword args for CSVFiles [here](https://github.com/queryverse/CSVFiles.jl/blob/153a21a1fef7245b3f4c27e64a6507e506b526f7/src/csv_writer.jl#L75).

Reply:
Now I've changed it.

Reply:
Great!